ADRs/RH/SIG-SRE/ADR-00007-Error-Budget-Policy.md

```diff
@@ -0,0 +1,248 @@
+
+
+## Proposed Architecture
+
+The proposed content is as follows:
+
+***
+
+# Error Budget Policy (draft)
+
+Service Name: \<Service/s Name\>
+
+# Service
+
+\<Briefly describe the service/s under the scope of this document.\>
+
+# Definitions
+
+* _SLI_: The Service Level Indicator is the carefully-considered measure for an aspect of the service that is considered necessary to its function (e.g. the login screen should be available).
+* _SLO_: The Service Level Objective is the desired target for the aspect measured by an SLI (e.g. the login screen should be available for 99% of the time).
+* _Error Budget_:  The Error budget is one of the fundamental metrics for service reliability. It is the measure of how much a service can be unreliable (e.g. fail) without impacting customers. Error budgets are always based on SLOs.
+* _SLO miss_: The state where the error budget has been consumed, and the level of service is not acceptable to the service owner. 
+* _Burn rate_: The speed you are consuming your error budget. For example, consuming 100% of the error budget in the expected period (for 30 days period - 30 days), consuming 200% of the error budget in the expected period (for 30 days period - 15 days). This should be used for SLO-based alerting. 
+* _Reliability Work_: The work that is needed to recover the service from an "SLO miss" state.
+* _Feature Work_: The work that is done to provide the features and functionality that is needed by that service.
+
+# Aim
+
+There are many inputs which affect the prioritization of work. The reliability of a service is a significant, but not the only, input to this process. The Error Budget is the primary metric that is used to inform the decision makers of issues that affect the reliability of a service, and by inference, the satisfaction of the customer with that service.
+
+The questions that this is aiming to help answer is:
+
+* "What do we need to do now that the error budget is all gone?"
+* "How do we detect the early signs of system degradation, and what automated mechanisms do we have in place to intercept them?"
+
+# Scope
+
+The scope of this policy is limited to: \<service\>.
+
+The SLOs for this \<service\> are defined \<here\> (link to SLO definitions document).
+
+# Out-of-Scope
+
+The scope of this policy does not apply to: \<service/aspect-of-service\>.
+
+# Current Status
+
+The SLOs for this service are tracked via \<this\> dashboard (link to dashboard or visualization of SLOs).
+
+# Decision Making Principles
+
+## Short-term
+
+* If the service has any remaining error budget for each SLO, releases (feature work) can proceed.
+* If the service has consumed all the error budget for at least one SLO, all releases will stop except for security fixes and the work needed to recover from the "SLO miss" state (reliability work).
```

Comment:
I think that could be changed to "overused" or "consumed more than the allowed error budget".
The rationale is that teams may want to explicitly burn remaining error budget before it "expires" by e.g. running chaos tests or other experiments.

Reply:
Great idea thanks @pilhuhn . I have made that change.

Reply:
Not sure I see it. Do you mean the "at the discretion .." part below? It would not be clear from this to me.

Reply:
Hey @pilhuhn . I changed the text to (commit [here](https://github.com/operate-first/sre/pull/23/commits/76955c2a88ec7c1d2c83a40c6935088ed6fad0a5)):

_If the service has consumed **more than** the **allowed** error budget for at least one SLO, all releases will stop except for security fixes and the work needed to recover from the "SLO miss" state (reliability work)._

Does that work for you?